### PR TITLE
Fix update_install test

### PR DESCRIPTION
**Pull request by DeepthiYV**
1. The test installs "cargo" and "rls" even though it should not  .  I modified the test to install only rust package and avoiding the installation of cago/rls.
Test is now failing in the rust installation due to issue : https://bugzilla.suse.com/show_bug.cgi?id=1200499

2.  Added openssl-1_1 and openssl-1_0_0 to the conflicts list.

- Related ticket:
    1. https://progress.opensuse.org/issues/112277
    2. https://progress.opensuse.org/issues/112376
- Needles: NO
- Verification run:  
 1. rust test - http://10.161.229.147/tests/3632#step/update_install/43 
 2. openssl  - http://10.161.229.147/tests/3638#step/update_install/38

### CODE REVIEW

File: tests/qam-updinstall/update_install.pm [hunk 1]
```diff
@@ -51,7 +51,11 @@ sub has_conflict {
         'python3-ldb' => 'python-ldb',
         'chrony-pool-suse' => 'chrony-pool-empty',
         libGLwM1 => 'libGLw1',
-        libiterm1 => 'terminfo-iterm'
+        libiterm1 => 'terminfo-iterm',
+        rust => 'rls',
+        'rust-gdb' => 'cargo',
+        'openssl-1_0_0' => 'openssl-1_1 '
```
[the remark this answers is not in the document]
Comment (DeepthiYV) [reply tests/qam-updinstall/update_install.pm:57]: Ok Sure :)

**Review by foursixnine — APPROVED** [review]

**Comment by punkioudi** [comment]
The VR 1 is failing with `the to be installed rust-1.60.0-150300.21.23.1.x86_64 requires 'rust1.60', but this requirement cannot be provided`, is this expected?

**Comment by DeepthiYV** [comment]
> The VR 1 is failing with `the to be installed rust-1.60.0-150300.21.23.1.x86_64 requires 'rust1.60', but this requirement cannot be provided`, is this expected?
 
I had raised bug for this : https://bugzilla.suse.com/show_bug.cgi?id=1200499

**Comment by punkioudi** [comment]
> > The VR 1 is failing with `the to be installed rust-1.60.0-150300.21.23.1.x86_64 requires 'rust1.60', but this requirement cannot be provided`, is this expected?
> 
> I had raised bug for this : https://bugzilla.suse.com/show_bug.cgi?id=1200499

Is this a regression, have you seen if an update is causing it? If yes, contact the openqa-reviewers :)

**Review by punkioudi — APPROVED** [review]

**Pull request merged**